index.html

```diff
@@ -29,22 +29,24 @@
 
 <body>
   <header>
-    <img src="logo.png" width="200px" alt="HackNext logo"/>
+    <img src="logo.png" width="200px" alt="HackNext logo" />
   </header>
 
   <main>
     <h2>what</h2>
     <p>a cabin-hack in the middle of Sweden</p>
-    <p>since 2014 a small self-organizing hack has taken place in Järvsö, groups of friends have met to hack on hobby projects, socialize, and eat well!</p>
+    <p>since 2014 a small self-organizing hack has taken place in Järvsö, groups of friends have met to hack on hobby
+      projects, socialize, and eat well!</p>
 
     <h2>dates</h2>
-    <p>2024-11-14</p>
+    <p>2025-11-20</p>
     <p>to</p>
-    <p>2024-11-17</p>
-    <a href="https://calendar.google.com/calendar/r/eventedit?text=Hack+Next+2024&dates=20241114/20241118&trp=false&details=https://hacknextse.github.io/&pli=1&sf=true&location=Alpstigen%202,+Alpstigen,+Jarvs%C3%B6,+Sverige&sprop=https://hacknextse.github.io/" target="_blank">
+    <p>2025-11-23</p>
+    <a href="https://calendar.google.com/calendar/r/eventedit?text=Hack+Next+2025&dates=20251120/20251123&trp=false&details=https://hacknextse.github.io/&pli=1&sf=true&location=Alpstigen%202,+Alpstigen,+Jarvs%C3%B6,+Sverige&sprop=https://hacknextse.github.io/"
```

Comment:
You almost got it all right, the end date in this link should be 20251124, not 20251123. Totally logical...

Reply:
ah same as with the ical then!